src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/integration/health/HealthCheckTest.kt

```diff
@@ -11,17 +13,23 @@ class HealthCheckTest : IntegrationTestBase() {
 
   @Test
   fun `Health page reports ok`() {
+    stubPing(200)
+
     webTestClient.get()
       .uri("/health")
       .exchange()
       .expectStatus()
       .isOk
       .expectBody()
       .jsonPath("status").isEqualTo("UP")
+      .jsonPath("components.hmppsAuthApiHealth.details.HttpStatus").isEqualTo("OK")
+      .jsonPath("components.prisonApiHealth.details.HttpStatus").isEqualTo("OK")
```

Comment:
not related to this but could do with adding a basm health check - I'll add a ticket